haiku-apps/bamkeys/bamkeys-1.0.0.recipe

```diff
@@ -27,7 +27,8 @@ change these settings you can - \
 1. Tab until the screen is selected. \
 2. Press the space-bar. A zone will darken in the grid. \
 3. You can use the arrow keys to change the selected zone. \
-4. Pressing (and releasing) a keyboard shortcut will set the hotkey for that zone. \
+4. Pressing (and releasing) a keyboard shortcut will 
+set the hotkey for that zone. \
```

Comment:
At the 80 char limit, you'll have to escape the linebreak with a "\", just like you did - wrongly - for the items 1 to 3. :)